utils/util_carto.py

```diff
@@ -14,21 +14,40 @@
 CARTO_USER = os.getenv('CARTO_WRI_RW_USER')
 CARTO_KEY = os.getenv('CARTO_WRI_RW_KEY')
 
-def upload_to_carto(file, privacy, collision_strategy='skip'):
+def upload_to_carto(file, privacy, *tags:str, collision_strategy='skip'):
     '''
     Upload tables to Carto
     INPUT   file: location of file on local computer that you want to upload (string)
             privacy: the privacy setting of the dataset to upload to Carto (string)
+            tags: add one or more tags(string) to the dataset, default is "rw"
```

Comment:
is this accurate?
```suggestion
            tags: one or more tags to add to the dataset on Carto (list of strings)
```

Reply:
Yes it needs a list of strings to add the tags.